packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1631,6 +1681,8 @@ class Package {
       a.pythonName.localeCompare(b.pythonName),
     );
 
+    const scripts: string[] = [];
```

Comment:
We tend to prefer this form for some reason:

```suggestion
    const scripts = new Array<string>();
```

Reply:
I just to try keep style of this file. Such structure appear 5 times in this script.